core/cmd/shell_local.go

```diff
@@ -675,10 +672,14 @@ func (s *Shell) RebroadcastTransactions(c *cli.Context) (err error) {
 	// TODO: BCF-2511 once the dust settles on BCF-2440/1 evaluate how the
 	// [loop.Relayer] interface needs to be extended to support programming similar to
 	// this pattern but in a chain-agnostic way
-	chain, err := app.GetRelayers().LegacyEVMChains().Get(chainID.String())
+	chainService, err := app.GetRelayers().LegacyEVMChains().Get(chainID.String())
 	if err != nil {
 		return s.errorOut(err)
 	}
+	chain, ok := chainService.(legacyevm.Chain)
```

Comment:
These were reduced to the common ChainService interface, so that we don't presume incompatibility. Then a cast is required for the extra methods that will only succeed if embedded, not a LOOPP.